vendor/github.com/go-openapi/errors/schema.go

```diff
@@ -65,6 +65,32 @@ const (
 	failedAllPatternPropsNoIn = "%s.%s failed all pattern properties"
 )
 
+// All code responses can be used to differentiate errors for different handling
+// by the consuming program
+const (
+	// CompositeErrorCode remains 422 for backwards-compatibility
+	// and to separate it from validation errors with cause
+	CompositeErrorCode = 422
+	// InvalidTypeCode is used for any subclass of invalid types
+	InvalidTypeCode = 600 + iota
+	RequiredFailCode
+	TooLongFailCode
+	TooShortFailCode
+	PatternFailCode
+	EnumFailCode
+	MultipleOfFailCode
+	MaxFailCode
```

Comment:
yes codes different from http status codes for errors make a lot of sense.
Not everything is string based, like translated error messages in a ui.